lib/puppet/configurer/plugin_handler.rb

```diff
@@ -2,6 +2,7 @@
 # just included into the agent, but having it here makes it
 # easier to test.
 require 'puppet/configurer'
+INTERNATIONALIZED_AGENT_VERSION = Gem::Version.new("5.3.4")
```

Comment:
We probably want to declare this constant inside the class since we only need it inside the class.

The name is also miss leading we support internationalized agents earlier than 5.3.4. How about
SUPPORTED_LOCALES_MOUNT_AGENT_VERSION